rastervision/protos/scene.proto

```diff
@@ -22,5 +22,9 @@ message SceneConfig {
     optional LabelStoreConfig prediction_label_store = 4;
 
     // Optional AOI that describes where the scene is fully labeled.
+
+    // XXX here for backward compatibility
     optional string aoi_uri = 5;
```

Comment:
XXX [may not be called for](https://softwareengineering.stackexchange.com/questions/65467/what-does-xxx-mean-in-a-comment) here.

Reply:
True, I took it out.